docs/streams/developer-guide/config-streams.html

```diff
@@ -742,7 +742,12 @@ <h4><a class="toc-backref" href="#id30">probing.rebalance.interval.ms</a><a clas
               and <code class="docutils literal"><span class="pre">transaction.state.log.min.isr</span></code>
               to the number of brokers you want to use.
               For more details see <a href="../core-concepts#streams_processing_guarantee">Processing Guarantees</a>.
-            </div></blockquote>
+
+              <dl class="docutils">
+                <dt>Recommendation:</dt>
+                  <dd>While it is technically possible to use EOS with any replication factor, using a replication factor lower than 3 effectively voids EOS. Thus it is strongly recommended to use a replication factor of 3 (together with min.in.sync.replicas=2). This recommendation applies to all topics (i.e. __transaction_state, __consumer_offsets, Kafka Streams internal topics, and user topics).</dd>
```

Comment:
Can we break this line into multiple? Hard to review long lines...

Should we use some markup for `__consumer_offsets` (etc) ?